src/components/assets/css/Signin.css

```diff
@@ -0,0 +1,67 @@
+h2 { 
+  width:100%; 
+  text-align:center; 
+  border-bottom: 5px solid gray; 
+  line-height:0.1em; 
+  margin:10px 0 20px;
+} 
+h2 span { 
+  background:#fff; 
+  padding:0 10px; 
+}
+.col {
+  text-align: center;
+  width: 100%;
+  margin: auto;
+  padding: 0 50px;
+  margin-top: 6px;
+  justify-content: space-around;
+}
+.col li{
+  margin:10px;
+}
+.col a{
+  width: 90%;
+}
+.fb {
+  background-color: #3B5998;
+  color: white;
+}
+.github{
+  background-color:black;
+  color:white;
+}
+.github-li :hover{
```

Comment:
The styled component for social media link will cover each of this. For hover refer - https://styled-components.com/docs/basics